### PR TITLE
[now-cli] Improve formatting of new deploy command prompts

### PR DESCRIPTION
**Pull request by lucleray**
- Adjust error prompt
- Add spinner while loading org and project
- Use lighter gray for infos
- Remove `{` in inspector url (was mistakenly added in https://github.com/zeit/now/pull/3599)
- Show error when entering a project name
- Fix bug with spinner not cleared when error is displayed
- "Finalizing" -> "Completing"

image

### CODE REVIEW

File: packages/now-cli/src/util/input/input-project.ts [hunk 1]
```diff
@@ -93,13 +101,22 @@ export default async function inputProject(
     });
     newProjectName = answers.newProjectName as string;
 
+    if (!newProjectName) {
+      output.print(`${chalk.red('Error!')} Project name can not be empty\n`);
```
Comment (AndyBitz) [inline packages/now-cli/src/util/input/input-project.ts]:
```suggestion
      output.print(`${chalk.red('Error!')} Project name cannot be empty\n`);
```

Comment (lucleray) [reply packages/now-cli/src/util/input/input-project.ts]: TIL https://www.dailywritingtips.com/cannot-or-can-not/

Comment (lucleray) [reply packages/now-cli/src/util/input/input-project.ts]: > The experts at AskOxford seem to prefer cannot

If the experts say it!

Comment (lucleray) [reply packages/now-cli/src/util/input/input-project.ts]: Thanks for the suggestion @AndyBitz 😎

File: packages/now-cli/src/util/input/input-project.ts [hunk 2]
```diff
@@ -69,9 +69,17 @@ export default async function inputProject(
       });
       const projectName = answers.existingProjectName as string;
 
-      const loader = wait('Verifying project name…', 1000);
-      project = await getProjectByIdOrName(client, projectName, org.id);
-      loader();
+      if (!projectName) {
+        output.print(`${chalk.red('Error!')} Project name can not be empty\n`);
```
Comment (leo) [inline packages/now-cli/src/util/input/input-project.ts]:
```suggestion
        output.print(`${chalk.red('Error!')} Project name cannot be empty\n`);
```

**Review by leo — CHANGES_REQUESTED** [review]

File: packages/now-cli/src/util/input/input-project.ts [hunk 3]
```diff
@@ -69,9 +69,17 @@ export default async function inputProject(
       });
       const projectName = answers.existingProjectName as string;
 
-      const loader = wait('Verifying project name…', 1000);
-      project = await getProjectByIdOrName(client, projectName, org.id);
-      loader();
+      if (!projectName) {
+        output.print(`${chalk.red('Error!')} Project name cannot be empty\n`);
```
Comment (leo) [inline packages/now-cli/src/util/input/input-project.ts]: Don't we have a util for printing errors?

Comment (AndyBitz) [reply packages/now-cli/src/util/input/input-project.ts]: Yea, `output.error` should work the same way

Comment (lucleray) [reply packages/now-cli/src/util/input/input-project.ts]: Good point!

```suggestion
        output.error(`Project name cannot be empty`);
```

File: packages/now-cli/src/util/input/input-project.ts [hunk 4]
```diff
@@ -93,13 +101,22 @@ export default async function inputProject(
     });
     newProjectName = answers.newProjectName as string;
 
+    if (!newProjectName) {
+      output.print(`${chalk.red('Error!')} Project name cannot be empty\n`);
```
Comment (lucleray) [inline packages/now-cli/src/util/input/input-project.ts]:
```suggestion
      output.error(`Project name cannot be empty`);
```

File: packages/now-cli/src/util/output/create-output.ts [hunk 5]
```diff
@@ -52,9 +52,9 @@ export default function createOutput({ debug: debugEnabled = false } = {}) {
   }
 
   function error(str: string, slug: string | null = null) {
-    log(chalk`{red.bold Error!} ${str}`, chalk.red);
+    print(`${chalk.red(`Error!`)} ${str}\n`);
     if (slug !== null) {
-      log(`More details: https://err.sh/now/${slug}`);
+      print(`More details: https://err.sh/now/${slug}\n`);
     }
   }
```
Comment (AndyBitz) [inline packages/now-cli/src/util/output/create-output.ts:59]: @lucleray Why was this changed? The output stays the same, except for "Error!" not being bold, right?

Comment (lucleray) [reply packages/now-cli/src/util/output/create-output.ts:59]: It removes the `>` in front of errors:
```
Before:
> Error! abcdef
After:
Error! abcdef
```

Comment (AndyBitz) [reply packages/now-cli/src/util/output/create-output.ts:59]: Oh, I see, because `log` adds the `>`, got it

**Review by AndyBitz — APPROVED** [review]

**Review by leo — APPROVED** [review]
I'd love for `log` to not have the `>` anymore, but I like your approach of tackling it incrementally, so that we're not breaking a UI that we're not considering.
